src/server/middleware/reactApplication/index.js

```diff
@@ -54,6 +55,9 @@ function reactApplicationMiddleware(request, response) {
     </CodeSplitProvider>,
   );
 
+  // render styled-components styleSheets to string.
+  const styles = styleSheet.rules().map(rule => rule.cssText).join('\n')
```

Comment:
You're having a string called `styles`, which has the stringified styles. So far, so good.